kernel/fs/initramfs.rs

```diff
@@ -52,6 +52,10 @@ impl FileLike for InitramFsFile {
     fn stat(&self) -> Result<Stat> {
         Ok(self.stat)
     }
+
```

Comment:
Please revert changes in this file. Let's work on chmod implementation in initramfs in a separate PR.